vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_monthly.cc

```diff
@@ -22,93 +23,86 @@ ContributionMonthly::ContributionMonthly(LedgerImpl* ledger) :
 
 ContributionMonthly::~ContributionMonthly() = default;
 
-void ContributionMonthly::Process(ledger::LegacyResultCallback callback) {
-  auto get_callback = std::bind(&ContributionMonthly::PrepareTipList,
-      this,
-      _1,
-      callback);
-
-  ledger_->contribution()->GetRecurringTips(get_callback);
+void ContributionMonthly::Process(base::Time cutoff_time,
+                                  ledger::LegacyResultCallback callback) {
+  ledger_->contribution()->GetRecurringTips(
+      [this, cutoff_time, callback](std::vector<mojom::PublisherInfoPtr> list) {
+        PrepareTipList(cutoff_time, callback, std::move(list));
+      });
 }
 
 void ContributionMonthly::PrepareTipList(
+    base::Time cutoff_time,
+    ledger::LegacyResultCallback callback,
+    std::vector<mojom::PublisherInfoPtr> list) {
```

Comment:
I'd name this variable differently, as we have actual list types (e.g. `std::list<>`). 🙂

Reply:
Good call!